client/packages/host/src/Help/FeedbackSection.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  DetailContainer,
+  Grid,
+  Typography,
+  useTranslation,
+} from '@openmsupply-client/common';
+import React from 'react';
+import { FeedbackForm } from './FeedbackForm';
+
+export const FeedbackSection = () => {
+  const t = useTranslation();
+  return (
+    <DetailContainer>
+      <Grid
+        display="flex"
+        flex={1}
+        flexDirection="column"
+        container
+        paddingTop={2}
+        paddingBottom={1}
```

Comment:
It looks like, between the padding here, and the padding defined the the `Help` component, we've probably got a bit too much padding. However, we're going to merge this with Aimee's docs link, so the padding might actually work out great... so let's leave for now, and we can review UI clean ups at the end of the feature if needed 🚀